index.html

```diff
@@ -218,17 +218,17 @@ <h3>CSS</h3>
         <div class="frameworks">
             <div id="intro2">
                 <h3>Frameworks</h3>
-                <button class="arrow2"><img src="images\Union@2x(1).png"></button>
+                <button class="arrow2"><img src="images\Union@2x(1).png" alt="Arrow"></button>
         </div>
     </div>
-        <div id="separation"><img scr="images\Separator Bottom@2x.png"></div>
+        <div id="separation"><img scr="images\Separator Bottom@2x.png" alt="Separation"></div>
         <div class="skills">
             <div id="intro3">
                 <h3>Skills</h3>
-                <button class="arrow2"><img src="images\Union@2x(1).png"></button>
+                <button class="arrow2"><img src="images\Union@2x(1).png" alt="Arrow"></button>
         </div>
     </div>
-        <div id="separation"><img scr="images\Separator Bottom@2x.png"></div>
+        <div id="separation"><img scr="images\Separator Bottom@2x.png" alt="Separation"></div>
```

Comment:
[OPTIONAL] Kindly fix the `scr` to `src` like you did on the other imgs. Currently, your image was broken.
Line 224 and 231. 👍
<img width="236" alt="Screenshot 2023-01-31 at 11 43 57 PM" src="https://user-images.githubusercontent.com/25887253/215878850-460a152b-1570-4c55-aa78-8d3c16e4fbc8.png">